src/notification.h

```diff
@@ -2,7 +2,9 @@
 #ifndef DUNST_NOTIFICATION_H
 #define DUNST_NOTIFICATION_H
 
-#include "x.h"
+#include <glib.h>         // for gsize
```

Comment:
No need to comment what each include is for, if someone removes one that's needed the compiler will complain anyway.